### ISSUE TITLE
Iconfinder blocked by CORS/CORP

### ISSUE THREAD

**Issue by Blfrg** [jgraph/drawio#3465]
### Preflight Checklist

* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a ~feature request~ _bug_ that matches the one I want to file, without success.

**Describe the bug**
I created a diagram within the last couple of months which I now know was using some images from iconfinder.com.
I opened the diagram yesterday, and some images no longer appear.
Searching for this icon now returns blank results.

**To Reproduce**
1. Go to https://app.diagrams.net/
2. Click on 'Search Shapes'
3. Search for 'QR'
4. See result tiles are blank
5. Check Developer Tools and see an `HTTP 403` response due to the [CORP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy) issue

**Expected behavior**
The images should be visible within the UI and diagram, especially if previously visible.

---

**Screenshots**

> Dev Tools reporting [CORP](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy) issue

![2023-03-26_09-55](https://user-images.githubusercontent.com/3829034/227789371-5110fca1-05c7-4666-aa78-bd7f3a42e369.png)

---

> Search for 'QR' showing blank response tiles

![2023-03-26_10-09](https://user-images.githubusercontent.com/3829034/227789372-c9281941-2978-4c69-9134-c9a6922872d3.png)

---

**draw.io version (In the Help->About menu of the draw.io editor):**

- draw.io version 21.1.1

**Desktop (please complete the following information):**
 - OS: Ubuntu 22.04
 - Browser: Chrome
 - Version: 111.0.5563.110 (Latest)

**Smartphone (please complete the following information):**
 - Device: N/A
 - OS: N/A
 - Browser: N/A
 - Version: N/A

**Additional context**
I understand that this may be outside of the control of this software.
If the issue cannot be resolved with the provider, then it should be removed, and hopefully, an alternative could replace it.

**Comment by davidjgraph** [comment]
Any browser extensions? Does it still happen incognito with extensions disabled?

**Comment by Blfrg** [comment]
Thank you for the prompt reply @davidjgraph 

When I reported the issue, I tested it in multiple browsers, without extensions, and even the desktop client, all with the same result.

Yesterday it appeared Iconfinder was having some [issues](https://iconfinder.statuspage.io/), so I waited a day.
But the problem now appears to have been resolved.

**Comment by davidjgraph** [comment]
Yup, seems iconfinder had an outage of some kind. Always good to see status pages accurately reflecting problems...
